src/main.ts

```
@@ -2384,19 +2384,33 @@ const ensureMarksWatcher = () => {
 
 ipcMain.handle('get-session-marks', () => {
   ensureMarksWatcher();
-  return readSessionMarks();
+  // `known` travels with the marks: an unreadable store yields empty marks
+  // that must not be mistaken for "this user has no pins" (see MarksRead).
+  const read = readSessionMarksResult();
+  return { ...read.marks, known: read.known };
 });
 
 ipcMain.handle('pin-session', (_event, sessionId: string, info: { cwd?: string; accountLabel?: string }) => {
```

ESLint

[error] Replace `'pin-session',·(_event,·sessionId:·string,·info:·{·cwd?:·string;·accountLabel?:·string·}` with `⏎··'pin-session',⏎··(⏎····_event,⏎····sessionId:·string,⏎····info:·{·cwd?:·string;·accountLabel?:·string·},⏎··`

(prettier/prettier)